.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,51 @@
+name: Test noir-wasm-testing
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
In `noir-lang/noir` we run tests on `[push, pull_request]`. These tests are pretty light so I think we can do the same here.

We want to at least run the tests on master as well.